sentry-core/src/main/java/io/sentry/core/UncaughtExceptionHandlerIntegration.java

```diff
@@ -58,8 +62,10 @@ public void uncaughtException(Thread thread, Throwable thrown) {
     logIfNotNull(options.getLogger(), SentryLevel.INFO, "Uncaught exception received.");
 
     try {
-      // TODO: Set Thread info to the scope?
-      this.hub.captureException(thrown);
+      Throwable throwable = getUnhandledThrowable(thread, thrown);
+      this.hub.captureException(throwable);
+      // Close the SDK to flush the event to disk before shutting down.
+      this.hub.close();
```

Comment:
right now `flush` is not implemented but getting `options.getShutdownTimeout()`
`connection.close();` awaits termination for 1 minute, should `close` also respect `options.getShutdownTimeout()`?